Makefile

```diff
@@ -33,11 +33,16 @@ endif
 .PHONY: test
 test:
 	# test all packages
-	GO111MODULE=on go test -coverprofile=coverage.txt -covermode=atomic -parallel 8 -race -coverpkg $(COVERPKGS) ./...
+	GO111MODULE=on MallocNanoZone=0 go test -coverprofile=coverage.txt -covermode=atomic -parallel 8 -race -coverpkg $(COVERPKGS) ./...
 	# remove coverage of empty functions from report
 	sed -i -e 's/^.* 0 0$$//' coverage.txt
 	cd ./languageserver && make test
 
+.PHONY: fast-test
```

Comment:
Good idea! This is what developers working on Cadence normally run locally; race detection, and coverage are much more CI features and add significant overhead 👍